RetailCoder.VBE/UI/Refactorings/ReorderParameters/ReorderParametersDialog.resx

```diff
@@ -0,0 +1,142 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we really want to mix and match XAML with Winforms? If we decide this dialog should be dockable later, this entire UI would need rebuilt and ported to Winforms. That may be unlikely, but something to consider.